FusionIIIT/applications/placement_cell/admin.py

```diff
@@ -7,28 +7,121 @@
+    list_display = ('unique_id', 'debar', 'future_aspect', 'placed_type', 'placement_date',
+                    'package')
+
+
+class MessageOfficerAdmin(admin.ModelAdmin):
+    fields = ['timestamp']
+
+
+class NotifyStudentAdmin(admin.ModelAdmin):
+    list_display = ('placement_type', 'company_name', 'ctc')
+
+
+class PlacementStatusAdmin(admin.ModelAdmin):
+    list_display = ('notify_id', 'unique_id', 'placed', 'timestamp')
+
+
+class PlacementRecordAdmin(admin.ModelAdmin):
+    list_display = ('placement_type', 'name', 'ctc', 'year', 'test_score', 'test_type')
+
+
+class StudentRecordAdmin(admin.ModelAdmin):
+    list_display = ('record_id', 'unique_id')
+
+
+class ChairmanVisitAdmin(admin.ModelAdmin):
+    list_display = ('company_name', 'location', 'visiting_date', 'timestamp')
+
+
+class ContactCompanyAdmin(admin.ModelAdmin):
+    list_display = ('company_name', 'hr_mail', 'timestamp')
+
+
+class PlacementScheduleAdmin(admin.ModelAdmin):
+    list_display = ('title', 'placement_date', 'location', 'time')
+
 
-admin.site.register(Project)
-admin.site.register(Language)
-admin.site.register(Know)
-admin.site.register(Skill)
-admin.site.register(Has)
-admin.site.register(Education)
-admin.site.register(Experience)
-admin.site.register(Course)
-admin.site.register(Publication)
-admin.site.register(Achievement)
-admin.site.register(Coauthor)
-admin.site.register(Patent)
-admin.site.register(Coinventor)
-admin.site.register(Interest)
-admin.site.register(StudentPlacement)
-admin.site.register(MessageOfficer)
-admin.site.register(NotifyStudent)
-admin.site.register(PlacementStatus)
-admin.site.register(PlacementRecord)
-admin.site.register(StudentRecord)
-admin.site.register(ChairmanVisit)
-admin.site.register(ContactCompany)
-admin.site.register(PlacementSchedule)
+admin.site.register(Project,  ProjectAdmin)
+admin.site.register(Language,  LanguageAdmin)
+admin.site.register(Know,  KnowAdmin)
+admin.site.register(Skill,  SkillAdmin)
+admin.site.register(Has,  HasAdmin)
+admin.site.register(Education,  EducationAdmin)
+admin.site.register(Experience,  ExperienceAdmin)
+admin.site.register(Course,  CourseAdmin)
+admin.site.register(Publication,  PublicationAdmin)
+admin.site.register(Achievement,  AchievementAdmin)
+admin.site.register(Coauthor,  CoauthorAdmin)
+admin.site.register(Patent,  PatentAdmin)
+admin.site.register(Coinventor,  CoinventorAdmin)
+admin.site.register(Interest,  InterestAdmin)
+admin.site.register(StudentPlacement,  StudentPlacementAdmin)
+admin.site.register(MessageOfficer, MessageOfficerAdmin)
+admin.site.register(NotifyStudent,  NotifyStudentAdmin)
+admin.site.register(PlacementStatus,  PlacementStatusAdmin)
+admin.site.register(PlacementRecord,  PlacementRecordAdmin)
+admin.site.register(StudentRecord,  StudentRecordAdmin)
+admin.site.register(ChairmanVisit, ChairmanVisitAdmin)
+admin.site.register(ContactCompany,  ContactCompanyAdmin)
+admin.site.register(PlacementSchedule,  PlacementScheduleAdmin)
```

Comment:
There is no problem here. I think there shouldn't be two spaces between arguments in `admin.site.register`  
I am merging the PR, please change this in your next PR.